v2-components/main-action-card/MainActionCard.tsx

```diff
@@ -0,0 +1,237 @@
+import { Badge, Box, Button, Flex, Heading, Stack, Text } from '@chakra-ui/react';
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { getVariant } from '../c-ratio-health-card/getVariant';
+import { InfoIcon } from '../icons';
+import { CollectIcon } from '../icons/CollectIcon/CollectIcon';
+import { MaintainIcon } from '../icons/MaintainIcon/MaintainIcon';
+import { StakeIcon } from '../icons/StakeIcon/StakeIcon';
+
+const CardHeader = ({
+  step,
+  headingText,
+  bodyText,
+  icon,
+}: PropsWithChildren<{ step: number; headingText: string; bodyText: string; icon: ReactNode }>) => {
+  return (
+    <Box>
+      <Flex alignItems={'center'} justifyContent={'space-between'}>
+        <Text fontSize="4xl" fontFamily="mono">
+          {step}
+        </Text>
+        {icon}
+      </Flex>
+      <Heading fontSize={'sm'}>{headingText}</Heading>
+      <Text color={'whiteAlpha.700'} fontSize={'xs'}>
+        {bodyText}
+      </Text>
+    </Box>
+  );
+};
+
+const Container = ({ children }: PropsWithChildren<{}>) => {
+  return (
+    <Flex
+      flexDirection="column"
+      justifyContent="space-between"
+      width={['full', '70%', '72']}
+      height={['auto', 'auto', '56']}
+      alignItems="space-between"
+      border={'1px'}
+      borderColor="gray.800"
+      padding="2"
+      borderRadius="sm"
+    >
+      {children}
+    </Flex>
+  );
+};
+
+const StakeActionCard: React.FC<Props> = ({ isStaking }) => {
+  return (
+    <Container>
+      <CardHeader
+        step={1}
+        headingText="Stake & Borrow"
+        bodyText="Mint sUSD by staking your SNX."
+        icon={<StakeIcon />}
+      />
+      {isStaking ? (
+        <Button
+          onClick={() => {
+            console.log('navigate to stake and borrow more');
+          }}
+          mb="2"
+          variant="link"
+        >
+          {'Stake & Borrow More'}
+        </Button>
+      ) : (
+        <Button
+          onClick={() => {
+            console.log('navigate to stake and borrow more');
+          }}
+          variant={'solid'}
+        >
+          Start Staking
+        </Button>
+      )}
+    </Container>
+  );
+};
+const MaintainActionCard: React.FC<Props & { isFlagged: boolean }> = ({
+  liquidationCratioPercentage,
+  targetCratioPercentage,
+  currentCRatioPercentage,
+  isStaking,
+  isFlagged,
+}) => {
+  const variant = getVariant({
+    liquidationCratioPercentage,
+    targetCratioPercentage,
+    currentCRatioPercentage,
+  });
+
+  return (
+    <Container>
+      <CardHeader
+        step={2}
+        headingText="Maintain Collateralization Ratio"
+        bodyText="Maintain your Collateralization Health."
+        icon={<MaintainIcon />}
+      />
+      {isStaking && (
+        <Badge
+          color={variant}
+          bg="blackAlpha.600"
+          border={'1px'}
+          borderColor={variant}
+          display="flex"
+          alignItems={'center'}
```

Comment:
It's a bit annoying but can we do just "center" and "1px"
My editor is code completing to {'center'} as well but bit annoying tbh

Reply:
yea I dont care too much. But if we wanna enforce it I'll add `'react/jsx-curly-brace-presence': ['warn', { props: 'never', children: 'never' }],`. Then eslint can fix all places for us. 78 places in total.. Probably wait with fixing that until migraiton to monorepo is done